lms/urls.py

```diff
@@ -61,6 +61,9 @@
 
     url(r'^heartbeat$', include('heartbeat.urls')),
 
+    # TODO jimabramson: put this in the right place
+    url(r'^api/profile_images/', include('openedx.core.djangoapps.profile_images.urls')),
```

Comment:
Actually, after further discussion, I'm OK with "profile_images" being its own package.

But take care of this "TODO".